### DISCUSSION TITLE
humming beat Ai

### DISCUSSION THREAD

**Discussion by tedted111** [community/community#141246] [Programming Help]
### Select Topic Area

Question

### Body

I want to make a song editor platform, but I want to put a humming function in it and make a platform that creates a beat that goes with it when you hum, unlike stable audio, when you hum a song in your head, it recommends a beat that goes with it, so I want to make an AI that makes the humming match the beat.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @tedted111 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Reply by tedted111** [reply]
감사합니다. 하지만 어떤 카테고리를 말씀하시는 건지 모르겠습니다. 알려주세용~~~!
